mkt/reviewers/tests/test_views.py

```diff
@@ -1300,21 +1301,39 @@ def test_public_sign_failure(self, sign_mock, json_mock,
 
 
 class TestReviewMixin(object):
+    # E.g commreply+12e0caffc4ca4174a6f62300c0ff180a@marketplace.firefox.com .
+    COMM_REPLY_RE = r'^commreply\+[a-f0-9]+\@marketplace\.firefox\.com$'
```

Comment:
you can use `re.compile()` to compile this so each time it is used it doesn't have to do it again.